src/main/java/dinkplugin/message/Field.java

```diff
@@ -20,6 +21,10 @@ public class Field {
     @Nullable Boolean inline;
 
     public Field(String name, String value) {
-        this(name, value, null);
+        this(name, value, true);
+    }
+
+    public static String format(String language, String value) {
```

Comment:
Maybe name `formatBlock` or `formatCodeBlock` or `formatValue` as that's how its most often used? and changing the name of the `language` parameter to something like `codeBlockLanguage` to clarify